src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/create/action/HandleNotCreatedFlowAction.java

```diff
@@ -43,7 +45,7 @@ public HandleNotCreatedFlowAction(PersistenceManager persistenceManager,
     public void perform(State from, State to, Event event, FlowCreateContext context, FlowCreateFsm stateMachine) {
         String flowId = stateMachine.getFlowId();
         dashboardLogger.onFlowStatusUpdate(flowId, FlowStatus.DOWN);
-        flowRepository.updateStatus(flowId, FlowStatus.DOWN);
+        flowRepository.updateStatus(flowId, FlowStatus.DOWN, format("Failed to create flow %s", flowId));
```

Comment:
Is the statusInfo supposed to bring light on "what goes wrong?"? There is 0 info about what goes wrong.

Reply:
To see what went wrong, we have history. When setting the task, I realized that this field is needed for filtering.